arangod/Aql/RestAqlHandler.cpp

```diff
@@ -275,7 +309,8 @@ bool RestAqlHandler::registerSnippets(
     VPackSlice const snippetsSlice, VPackSlice const collectionSlice,
     VPackSlice const variablesSlice, std::shared_ptr<VPackBuilder> const& options,
     std::shared_ptr<transaction::Context> const& ctx, double const ttl,
-    SerializationFormat format, bool& needToLock, VPackBuilder& answerBuilder) {
+    SerializationFormat format, bool& needToLock, VPackBuilder& answerBuilder,
+    RebootId rebootId, std::string coordinatorId) {
```

Comment:
Can be passed by const reference, but this is an unimportant optimization here.
```suggestion
    RebootId rebootId, std::string const& coordinatorId) {
```

Reply:
done